search-app/docs/api.md

```diff
@@ -1781,7 +1789,15 @@ The parameters used for searching granules by spatial are the same as the spatia
 
 ##### <a name="g-line"></a> Line
 
-    curl "%CMR-ENDPOINT%/granules?collection_concept_id=%CMR-EXAMPLE-COLLECTION-ID%&line=-0.37,-14.07,4.75,1.27,25.13,-15.51"
+    curl "%CMR-ENDPOINT%/granules?collection_concept_id=%CMR-EXAMPLE-COLLECTION-ID%&line=-0.37,-14.07,4.75,1.27,25.13,-15.51"  
+    
+##### <a name="g-wkt"></a> Wkt
+
+Various spatial geometries can be represented using a subset of Well Known Text (WKT). The currently supported geometries include: point, line, and polygon.
```

Comment:
Same as with the collection documentation - just a little bit more detail would be nice